WinUIGallery/ControlPages/DesignGuidance/IconographyPage.xaml

```diff
@@ -206,6 +204,19 @@
                         <controls:SampleCodePresenter x:Name="XAMLCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
                         <TextBlock Text="C#" />
                         <controls:SampleCodePresenter x:Name="CSharpCodePresenter" Style="{StaticResource CodeValuePresenterStyle}" />
+                        <TextBlock Text="Tags" />
+                        <ItemsControl ItemsSource="{Binding SelectedItem.Tags, Mode=OneWay}" Margin="0,4,0,4">
```

Comment:
I noticed that when an icon has no tags, the area remains empty.
This might leave the user confused or unsure if the feature is working correctly.
![image](https://github.com/user-attachments/assets/c2bdd52f-8005-47dc-aeac-f9ce51ea6424)

I think it would improve the user experience if you displayed a default message in such cases, like "No tags available for this icon" or "This icon has no associated tags yet."
This way, the user gets clear feedback about what’s happening :)

Reply:
Thanks! I just added a commit for the message.